Source/Scene/Cesium3DTile.js

```diff
@@ -0,0 +1,1018 @@
+         *
+         * @type {Boolean}
+         *
+         * @private
+         */
+        this.selected = false;
+
+
+        /**
+         * The time when a style was last applied to this tile.
+         *
+         * @type {Number}
+         *
+         * @private
+         */
+        this.lastStyleTime = 0;
+
+        /**
+         * Marks whether the tile's children bounds are fully contained within the tile's bounds
+         *
+         * @type {Cesium3DTileOptimizationHint}
+         *
+         * @private
+         */
+        this._optimChildrenWithinParent = Cesium3DTileOptimizationHint.NOT_COMPUTED;
+
+        // Members that are updated every frame for tree traversal and rendering optimizations:
+        this._distanceToCamera = 0;
+        this._visibilityPlaneMask = 0;
+        this._childrenVisibility = Cesium3DTileChildrenVisibility.VISIBLE;
+        this._lastSelectedFrameNumber = -1;
+        this._screenSpaceError = 0;
+        this._screenSpaceErrorComputedFrame = -1;
+        this._finalResolution = true;
+        this._requestHeap = undefined;
+        this._depth = 0;
+        this._centerZDepth = 0;
+        this._stackLength = 0;
+        this._selectedFrame = -1;
+        this._selectionDepth = 0;
+        this._lastSelectionDepth = undefined;
+        this._requestedFrame = undefined;
+        this._lastVisitedFrame = undefined;
+        this._ancestorWithContent = undefined;
+        this._ancestorWithLoadedContent = undefined;
+
+        this._debugBoundingVolume = undefined;
+        this._debugContentBoundingVolume = undefined;
+        this._debugViewerRequestVolume = undefined;
+        this._debugColor = Color.fromRandom({ alpha : 1.0 });
+        this._debugColorizeTiles = false;
+
+        this._commandsLength = 0;
+
+        this._color = undefined;
+        this._colorDirty = false;
+    }
+
+    defineProperties(Cesium3DTile.prototype, {
+        /**
+         * The tileset containing this tile.
+         *
+         * @memberof Cesium3DTile.prototype
+         *
+         * @type {Cesium3DTileset}
+         * @readonly
+         */
+        tileset : {
+            get : function() {
+                return this._tileset;
+            }
+        },
+
+        /**
+         * The tile's content.  This represents the actual tile's payload,
+         * not the content's metadata in tileset.json.
+         *
+         * @memberof Cesium3DTile.prototype
+         *
+         * @type {Cesium3DTileContent}
+         * @readonly
+         */
+        content : {
+            get : function() {
+                return this._content;
+            }
+        },
+
+        /**
+         * Get the bounding volume of the tile's contents.  This defaults to the
+         * tile's bounding volume when the content's bounding volume is
+         * <code>undefined</code>.
+         *
+         * @memberof Cesium3DTile.prototype
+         *
+         * @type {TileBoundingVolume}
+         * @readonly
+         * @private
```

Comment:
This PR seems to have a ton of non-underscore private properties.  That could be pretty confusing going forward.  I'm not sure what the right solution is, but are we sure just using underscores isn't the right option?